packages/html/src/column-menu/column-menu-expander.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import { classNames } from '../misc';
+import { Icon } from '../icon';
+import { ColumnMenuItem, ColumnMenuItemContent } from '../column-menu';
+
+export const COLUMNMENUEXPANDER_CLASSNAME = `k-expander`;
+
+const states = [];
+
+const options = {};
+
+export type KendoColumnMenuExpanderProps = {
+    itemText?: string;
+    itemIcon?: string;
+    expanderIcon?: string;
+    itemContent?: JSX.Element;
+    expanded?: boolean;
+};
+
+const defaultProps = {};
+
+export const ColumnMenuExpander = (
+    props: KendoColumnMenuExpanderProps &
+        React.HTMLAttributes<HTMLDivElement>
+) => {
+    const {
+        itemText,
+        itemIcon,
+        expanded,
+        itemContent,
+        expanderIcon,
```

Comment:
Add `{...other}` otherwise you won't have access to props from `<HTMLDivElement>` if you need them.